process/transaction/baseProcess.go

```diff
@@ -202,15 +202,15 @@ func (txProc *baseTxProcessor) processIfTxErrorCrossShard(tx *transaction.Transa
 }
 
 // VerifyTransaction verifies the account states in respect with the transaction data
-func (txProc *txProcessor) VerifyTransaction(tx *transaction.Transaction) error {
+func (txProc *txProcessor) VerifyTransaction(tx *transaction.Transaction) (state.UserAccountHandler, error) {
```

Comment:
Please revert this. I know this is a slight optimization but it breaks the SRP (single responsibility principle) as it does 2 things now: it checks the transaction & returns an account

Reply:
Done